src/core/ContentWindowController.h

```diff
@@ -75,6 +75,9 @@ class ContentWindowController : public QObject
     /** Resize the window. */
     Q_INVOKABLE void resize( QSizeF size, const WindowPoint fixedPoint = TOP_LEFT );
 
+    /** Resize the window relative to the given border */
+    Q_INVOKABLE void resizeRelative( const QPointF delta, const int border );
```

Comment:
I wish too if I would know how to make that type usable in QML. It's complaining about "Component not ready"
Unit test will be added